src/compas_fab/backends/pybullet/backend_features/pybullet_inverse_kinematics.py

```diff
@@ -77,7 +77,9 @@ def inverse_kinematics(self, robot, frame_WCF, start_configuration=None, group=N
             start_configuration = self.client.set_robot_configuration(robot, start_configuration, group)
 
         called_from_test = 'pytest' in sys.modules
-        if options.get('enforce_joint_limits', True) and not called_from_test:
+        enforce_joint_limits = options.get('enforce_joint_limits') or True
```

Comment:
This one's no good at all.  It forces `enforce_joint_limits` to never be `False`. Maybe something like
```suggestion
        enforce_joint_limits = True if options.get('enforce_joint_limits') is None or options['enforce_joint_limits']
```